packages/block-library/src/site-logo/icon.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * WordPress dependencies
+ */
+import { SVG, Path } from '@wordpress/components';
+
+export default (
+
+	<SVG height="21" viewBox="0 0 20 21" width="20">
+		<Path d="m5 9.5h-5v2h5zm2.17-3.24-2.12-2.12-1.41 1.41 2.12 2.12zm3.83-5.76h-2v5h2zm5.36 5.05-1.41-1.41-2.12 2.12 1.41 1.41zm-1.36 3.95v2h5v-2zm-5-2c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3zm2.83 7.24 2.12 2.12 1.41-1.41-2.12-2.12zm-9.19.71 1.41 1.41 2.12-2.12-1.41-1.41zm5.36 5.05h2v-5h-2z" fill="#525d67" />
+	</SVG>
+);
```

Comment:
Do you this icon can be added to @wordpress/icons or it's not generic enough?

Reply:
Happy to add it if you think there's value in it. Probably best to do it in a followup PR.